src/blocks/gallery-carousel/edit.js

```diff
@@ -327,6 +365,32 @@ class GalleryCarouselEdit extends Component {
 						</div>
 					</div>
 				</ResizableBox>
+				<div className={ className }>
+					<div
+						className={ innerClasses }
+						style={ innerStyles }
+						style={ navStyles }
```

Comment:
I'm unsure which style var we should be using here. ESLint threw an error about duplicate props here.